### PR TITLE
chore: Remove obsolete build scripts

### PR DESCRIPTION
**Pull request by lucydodo**
We recently finished moving our build system to GitHub Actions, so I've been removing obsolete scripts.

### CODE REVIEW

**Comment by justinclift** [comment]
We'll probably want to keep the cron script that moves files into subdirectories automatically each month. :smile:

**Comment by lucydodo** [comment]
Good catch.
To add to that, I remove the code for the old `arm64`, `intel` builds and changed it to work for the universal builds.

**Comment by justinclift** [comment]
Good thinking.  I'd totally forgotten to make those changes to the actual cron script (a copy of this) on the nightlies server.

Hmmm, any interest in updating the script on the nightlies server too while you're at it? :smile:

File: installer/other/move_nightlies_into_dirs.sh [hunk 1]
```diff
@@ -9,19 +9,17 @@ YEARMONTH=`date -d "last month 13:00" '+%Y-%m'`
...
-mv /nightlies/macos-arm64/DB*${YEARMONTHOSX}* /nightlies/macos-arm64/night*${YEARMONTHOSX}* /nightlies/macos-arm64/${YEARMONTH}/
+mv /nightlies/macos-universal/DB*${YEARMONTHOSX}* /nightlies/macos-universal/night*${YEARMONTHOSX}* /nightlies/macos-universal/${YEARMONTH}/
 mv /nightlies/win32/DB*${YEARMONTH}* /nightlies/win32/${YEARMONTH}/
 mv /nightlies/win64/DB*${YEARMONTH}* /nightlies/win64/${YEARMONTH}/
 
 # Fix ownership and SELinux context's
-chown -Rh nightlies: /nightlies/macos-intel/${YEARMONTH} /nightlies/macos-arm64/${YEARMONTH} /nightlies/win32/${YEARMONTH} /nightlies/win64/${YEARMONTH}
-restorecon -RFv /nightlies/macos-intel/${YEARMONTH} /nightlies/macos-arm64/${YEARMONTH} /nightlies/win32/${YEARMONTH} /nightlies/win64/${YEARMONTH}
+chown -Rh nightlies: /nightlies/macos-universal/${YEARMONTH} /nightlies/win32/${YEARMONTH} /nightlies/win64/${YEARMONTH}
+restorecon -RFv /nightlies/macos-universal/${YEARMONTH} /nightlies/win32/${YEARMONTH} /nightlies/win64/${YEARMONTH}
```
Comment (justinclift) [inline installer/other/move_nightlies_into_dirs.sh:23]: We don't need `restorecon` in the script any more.  That command is specific to SELinux, and only useful for Red Hat based Linux distributions.

We've the nightlies server from CentOS to Ubuntu a fair while ago.  Looks like we forgot to update this script in our GitHub repository at the time. :angel:

Comment (lucydodo) [reply installer/other/move_nightlies_into_dirs.sh]: Okay, I've removed that code.

**Comment by lucydodo** [comment]
> Hmmm, any interest in updating the script on the nightlies server too while you're at it? 😄

Yes, I've modified it to match your review, so take a look and if you approve, I'll replace the script. :)

**Review by justinclift — APPROVED** [review]
Cool, looks good to me. :smile:

**Comment by lucydodo** [comment]
Thanks, I've also replaced it on the server.

**Pull request merged**